src/main/scala/com/databricks/labs/overwatch/pipeline/Initializer.scala

```diff
@@ -237,6 +237,43 @@ class Initializer(config: Config) extends SparkSessionWrapper {
     config.setTempWorkingDir(workspaceTempWorkingDir)
   }
 
+
+  private def validateApiEnvConfig(apiEnvConfig: Option[ApiEnvConfig]) = {
```

Comment:
please create valid ranges, not just ensuring it's >1 but it should be between some valid range. I mean the success batch size shouldn't be 200000000000 right, so just put in what you believe to be valid range

Reply:
sure, previously i thought of implementing it but i did not had any range to justify so removed it. Now i am adding it